packages/block-library/src/shortcode/edit.native.js

```diff
@@ -36,6 +36,9 @@ export function ShortcodeEdit( props ) {
 				isSelected={ props.isSelected }
 				onFocus={ onFocus }
 				onBlur={ onBlur }
+				autoCorrect={ false }
+				autoComplete="off"
+				keyboardType={ Platform.OS === 'ios' ? 'default' : 'visible-password' }
```

Comment:
What's the reason for the `visible-password` here?

Reply:
In my testing, I found `autoCorrect={ false }` and `autoComplete="off"` only worked on some devices (not my pixel, for example), so I found [this post](https://stackoverflow.com/a/51411575/1350218) that suggested creating a specific class or adding this flag.

Reply:
Ah thanks. I don't love it, but if that's what it takes to make it work...
Could you add that as a comment in the code so it's clearer in the future?

Reply:
Thanks for adding the comment, and yep it's sad to have to do these kind of workaround :/.